oeqPrimaryWS/src/main/java/com/tle/blackboard/webservice/impl/ZEquellaWebserviceImpl.java

```diff
@@ -360,30 +345,13 @@ public SearchResult findAllUsages(@WebParam(mode = WebParam.Mode.IN, name = "ser
     } catch (WebserviceException ws) {
       throw ws;
     } catch (Exception t) {
-      chuckIt(t, "EQ011");
+      WsHelper.chuckIt(t, "EQ011");
       return null;
     } finally {
       Thread.currentThread().setContextClassLoader(oldLoader);
     }
   }
 
-  private Folder getFolder(BbWsSession session, Map<String, Folder> folderMap, Course course, String folderId)
-      throws PersistenceException {
-    Folder folder = folderMap.get(folderId);
-    if (folder == null) {
-      // WebServiceUtil.debug("Folder " + folderId +
-      // " not found in map, loading it");
-      Content bbFolder = session.loadContent(folderId);
-      if (bbFolder != null) {
-        folder = WebServiceUtil.convertFolder(bbFolder, course.getId());
-        folderMap.put(folderId, folder);
-      } else {
-        WebServiceUtil.debug("Folder could not be loaded.  It doesn't exist any more!");
-      }
-    }
-    return folder;
-  }
-
   @Override
   @WebMethod(action = "synchroniseEquellaContentTables")
```

Comment:
```suggestion
  @WebMethod(action = "synchronizeEquellaContentTables")
```

Reply:
We couldn't change this one even if we wanted to, it would break the web service contract.